sdk/communication/communication-phone-numbers/swagger/README.md

```diff
@@ -63,3 +63,24 @@ directive:
   transform: >
     $["x-ms-client-name"] = "AreaCodeItem";
 ```
+
+### Removed Property error from PhoneNumberSearchResult
+``` yaml
+directive:
+  where-model: PhoneNumberSearchResult
+  remove-property: error
+```
+
+### Removed Property error from PhoneNumberSearchResult
+``` yaml
+directive:
+  where-model: PhoneNumberSearchResult
+  remove-property: errorCode
+```
+
+### Removed Property phoneNumberSource from PurchasedPhoneNumber
+``` yaml
+directive:
+  where-model: PurchasedPhoneNumber
+  remove-property: phoneNumberSource
+```
```

Comment:
Could we add the same suppressions from the .NET SDK to prevent cherry picker generation? This is important because the generated client counts towards our coverage, but it is not used at all.

https://github.com/Azure/azure-sdk-for-net/blob/03d46a5e0a868fa42d4e25c2bd8065932a209e0e/sdk/communication/Azure.Communication.PhoneNumbers/src/autorest.md?plain=1#L44-L54